README.md

```diff
@@ -295,6 +295,22 @@ The following tools are required to test and run PolyTracker:
 * [gllvm](https://github.com/SRI-CSL/gllvm) (`go get github.com/SRI-CSL/gllvm/cmd/...`) is used to create whole program
   bitcode archives and to extract bitcode from targets.
 
+### Building on Apple silicon:
+
+Prebuilt Docker images for `polytracker-llvm` are only available for `amd64`. Users with `arm64` systems will have to
+build the image locally and then change `polytracker`'s Dockerfile to point to it:
+```commandline
+$ mkdir repos && cd repos
+$ git clone https://github.com/trailofbits/polytracker
+$ git clone https://github.com/trailofbits/polytracker-llvm
+$ cd polytracker-llvm
+$ DOCKER_BUILDKIT=1 docker build -t trailofbits/polytracker-llvm .
+$ cd ../polytracker
+$ ## Replace the tagged image name on the first line with trailofbits/polytracker:latest
```

Comment:
Suggest that we provide the full line e.g. `FROM trailofbits/polytracker:latest`

Reply:
Fixed! I also realised it should have said `polytracker-llvm` instead of `polytracker`